app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -343,6 +343,22 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testStateFragment_loadExp_secondState_submitAnswer_clickSubmit_isToolbarTitleSelected() {
+    launchForExploration(TEST_EXPLORATION_ID_2).use {
+      startPlayingExploration()
+      clickContinueInteractionButton()
+      typeFractionText("1/2")
+
+      clickSubmitAnswerButton()
+
+      scrollToViewType(CONTINUE_NAVIGATION_BUTTON)
```

Comment:
As you had described, you want to check `isSelected` after submit button click, but here you are checking after continue button click.